tests/test_cli.py

```diff
@@ -662,12 +677,28 @@ def test_genetic_map_citations(self, caplog):
         contig = species.get_contig("chr20", genetic_map=genetic_map.id)
         model = stdpopsim.PiecewiseConstantSize(species.population_size)
         engine = stdpopsim.get_default_engine()
+        dfe = None
         stdout, stderr = capture_output(
-            cli.write_citations, engine, model, contig, species
+            cli.write_citations, engine, model, contig, species, dfe
         )
         assert len(stdout) == 0
         self.check_citations(engine, species, genetic_map, model, caplog.text)
 
+    def test_dfe_citations(self, caplog):
```

Comment:
I think its hard to see where the `caplog` argument comes from (its a "magic" word that pytest recognises). To make this easier for future code readers, could we mark the test with the `@pytest.mark.usefixtures("caplog")` decorator?

Reply:
just putting this @pytest.mark.usefixtures("caplog") above the test?

Reply:
yep!